packages/components/src/components/icon/library.system.ts

```diff
@@ -60,7 +60,7 @@ export const icons = {
   `,
   'info-circle': `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2Zm0 18a8 8 0 1 1 0-16 8 8 0 0 1 0 16Z"/><path d="M13.333 7.833a1.5 1.5 0 1 1-3.001-.001 1.5 1.5 0 0 1 3.001.001ZM13 11.667a1 1 0 0 0-1-1h-1a1 1 0 0 0 0 2v2.667h2v-3.667Z"/><path d="M11 16.333v-1h-1a1 1 0 0 0 0 2h2a1 1 0 0 1-1-1Zm3-1h-1v1a1 1 0 0 1-1 1h2a1 1 0 0 0 0-2Z"/><path  d="M13 16.333v-1h-2v1a1 1 0 0 0 2 0Z"/></svg>`,
   'status-hook': `
-    <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="none">
+    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 12 13">
```

Comment:
Is 0 0 12 13 correct?

Reply:
@mariohamann I set those values manually since the checkmark path is not centered. it's 12 13 since the width is larger. I made sure with Van that this change leaves the other components that use the icon unchanged.

Reply:
Fine!